sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventParticipantsController.java

```diff
@@ -144,6 +144,15 @@ public EventParticipantDto createEventParticipant(
 							},
 							true);
 				} else {
+					EventParticipantCriteria criteria = new EventParticipantCriteria();
+					criteria.withEvent(eventRef);
+					List<EventParticipantIndexDto> currentEventParticipants =
+						FacadeProvider.getEventParticipantFacade().getIndexList(criteria, null, null, null);
```

Comment:
would be more performant to create a special backend method with an exists query for checking the person